### PR TITLE
Feat: Preparing to create Home page and Careers page

### PR DESCRIPTION
**Pull request by TangoPJ**
TODO:

1. prepare global styles
2. add sass module
3. add logic to dynamic document title
4. add Home and Careers components
5. fix routing

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-name: Build CI
```
Comment (ibotlold) [inline .github/workflows/pr.yml]: please do not make changes in pr workflow

File: package.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.3.0",
+    "react-router-dom": "^6.3.0",
```
Comment (ibotlold) [inline package.json]: Duplicate entry.
Pay attention to [my commit's comment](https://github.com/fakng-agrgtr/fakng-ui/commit/aff0307d3e62a33b602266a236a27b534bb0a76a#r81381299)

File: package.json [hunk 3]
```diff
@@ -43,5 +44,15 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "eslint": "^8.22.0",
```
Comment (ibotlold) [inline package.json]: eslint does not requires be linked in devDependencies cause react have built-in one

Comment (ibotlold) [reply package.json:49]: this was changed in #19

Comment (ibotlold) [reply package.json]: > eslint does not requires be linked in devDependencies cause react have built-in one

this was merged into master on #19

File: package.json (hunk 2 above)
Comment (TangoPJ) [reply package.json]: Done)

**Review by ibotlold — COMMENTED** [review]
a09effdceb03ddd332f725aaeca13ea9a94c5422 Here recommend use rebase instead of merging master branch

**Review by mighty-mhsl — COMMENTED** [review]
Please, push the fresh version of the code with all the below conversations resolved. Looks good so far, but need to resolve those comments and take the last look at it.

Thank you!

**Comment by ibotlold** [comment]
Really, cant wait to see what you guys done 👍

**Comment by ibotlold** [comment]
Resolved my comments

File: src/components/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  return (
+    <AppBar position="static">
+      <Container maxWidth="xl">
+        <Toolbar disableGutters>
+          <Typography
+            variant="h5"
+            noWrap
+            component="a"
+            href="/"
+            sx={{
```
Comment (ibotlold) [inline src/components/Header.jsx:25]: Styles needs to be extracted

File: src/App.js [hunk 5]
```diff
@@ -1,29 +1,44 @@
...
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <CssBaseline />
+      <Header />
+      {progress ? <LinearIndeterminate /> : (
+        <Routes>
+          <Route exact path="/" element={<Home tabTitle="Fakng - Home page" />} />
```
Comment (mighty-mhsl) [inline src/App.js:37]: I think we're good with just "Home page" etc, without additional words :)

Comment (ibotlold) [reply src/App.js:37]: let it stay that way. Its part for i18n job and can be done later

**Comment by ibotlold** [comment]
Provided basic components' test

File: src/components/Header.jsx (hunk 4 above)
Comment (ibotlold) [reply src/components/Header.jsx:25]: Can be resolved in future

**Review by mighty-mhsl — APPROVED** [review]

**Pull request merged**